src/modules/page/Login.page.unit.ts

```diff
@@ -0,0 +1,182 @@
+import {
+	createTestingI18n,
+	createTestingVuetify,
+} from "@@/tests/test-utils/setup";
+import { useAuthStore } from "@data/auth";
+import { createMock, DeepMocked } from "@golevelup/ts-jest";
+import { createTestingPinia } from "@pinia/testing";
+import { mount } from "@vue/test-utils";
+import { setActivePinia } from "pinia";
+import { Router, useRouter } from "vue-router";
+import { postLoginRoute } from "../../router/postLoginRoute";
```

Comment:
Maybe using the "@" alias is better for import here, like how it is imported in the vue component